### PR TITLE
Added top trending stacks and dependencies information

### PR DESCRIPTION
**Pull request by samuzzal-choudhury**

### CODE REVIEW

**Comment by samuzzal-choudhury** [comment]
@abs51295 I am adding unit tests for the newly added methods. Please wait for the upcoming commit on that.

**Comment by samuzzal-choudhury** [comment]
@abs51295 PTAL

File: f8a_report/report_helper.py [hunk 1]
```diff
@@ -519,6 +537,8 @@ def retrieve_worker_results(self, start_date, end_date, id_list=[], worker_list=
                                                         frequency)
         return result
 
+    '''
```
Comment (abs51295) [inline f8a_report/report_helper.py]: Can't we just remove this piece of code instead of commenting it?

File: f8a_report/report_helper.py [hunk 2]
```diff
@@ -670,11 +690,11 @@ def normalize_ingestion_data(self, start_date, end_date, ingestion_data, frequen
         except Exception as e:
             logger.exception('Unable to store the report on S3. Reason: %r' % e)
         return template
-
+    '''
     def get_report(self, start_date, end_date, frequency='daily'):
         """Generate the stacks report."""
         ids = self.retrieve_stack_analyses_ids(start_date, end_date)
-        ingestion_results = False
+        '''ingestion_results = False
```
Comment (abs51295) [inline f8a_report/report_helper.py]: Same here.

**Comment by samuzzal-choudhury** [comment]
> Some minor nitpicks.

@abs51295 done. Can you please have a relook at, approve and merge this PR when the build is through.

**Comment by samuzzal-choudhury** [comment]
Merging the PR

**Pull request merged**